build.js

```diff
@@ -4,30 +4,24 @@ const Ajv = require('ajv');
 const standaloneCode = require('ajv/dist/standalone').default;
 const { build } = require('esbuild');
 const replace = require('replace-in-file');
-const {
-  auctionBidSchema,
-  buyRecordSchema,
-  extendRecordSchema,
-  increaseUndernameCountSchema,
-} = require('./schemas');
+const schemas = require('./schemas');
+const { startCase } = require('lodash');
 
 // build our validation source code
 const ajv = new Ajv({
-  schemas: [
-    auctionBidSchema,
-    buyRecordSchema,
-    extendRecordSchema,
-    increaseUndernameCountSchema,
-  ],
+  schemas: Object.values(schemas), // use all exported schemas
   code: { source: true, esm: true },
   allErrors: true,
 });
 
+const definitions = Object.values(schemas).reduce((acc, schema) => {
+  const schemaName = schema['$id']?.split('/').pop();
+  acc['validate' + startCase(schemaName).split(' ').join('')] = schema.$id;
+  return acc;
```

Comment:
quite a bit of hackiness going on here

Reply:
I cleaned this up with a custom keyword for the validator function name.

Reply:
let's move these changes to a separate PR and keep this one scoped to contract API changes.